index.html

```diff
@@ -25,7 +25,7 @@
           </section>
         </aside>
       </div>
-      <div id = "legend-house">
+      <div id = "legend-house" data-city = "Lviv" data-street = "Staroyevreyska" data-number-street = "48" data-zip-code = "79000">
```

Comment:
Ok, now let's rename street number attribute into `data-street-number` and wrap this PR up.